### PR TITLE
Restore PROMETHEUS_SCRAPE_KUBE_PROXY guarding

### PR DESCRIPTION
**Pull request by jkaniuk**
Needed back as on GKE kube-proxy listens only on localhost

/cc wojtek-t
/cc mm4tt
/cc oxddr

### CODE REVIEW

File: clusterloader2/testing/load/experimental/overrides/disable_kubeproxy.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+PROMETHEUS_SCRAPE_KUBE_PROXY: false
```
Comment (mm4tt) [inline clusterloader2/testing/load/experimental/overrides/disable_kubeproxy.yaml]: NIT: Rename to include prometheus or scrape, as we're not disabling kube-proxy but scraping kube-proxy

Comment (mm4tt) [reply clusterloader2/testing/load/experimental/overrides/disable_kubeproxy.yaml]: Also move it outside experimental directory, I think there is a dir with prometheus overrides

File: clusterloader2/testing/load/config.yaml [hunk 2]
```diff
@@ -93,10 +94,12 @@ steps:
     Params:
       action: start
       replicasPerProbe: {{AddInt 2 (DivideInt .Nodes 100)}}
+  {{if $PROMETHEUS_SCRAPE_KUBE_PROXY}}
```
Comment (wojtek-t) [inline clusterloader2/testing/load/config.yaml]: Given that this measurement has no way to work if scraping kube-proxy is disabled, we should simply encode that in the measurement itself (it should be checking if we're scriaping kube-proxy and self-disable if we're not).

Comment (mm4tt) [reply clusterloader2/testing/load/config.yaml]: I agree, created https://github.com/kubernetes/perf-tests/issues/807 for that.

Comment (jkaniuk) [reply clusterloader2/testing/load/config.yaml]: Let's do it in another commit.
(or find some solution to get that metric)

File: clusterloader2/testing/load/experimental/overrides/disable_kubeproxy.yaml (hunk 1 above)
Comment (jkaniuk) [reply clusterloader2/testing/load/experimental/overrides/disable_kubeproxy.yaml]: Done

**Pull request merged**